### ISSUE TITLE
compilation warnings with ifort

### ISSUE THREAD

**Issue by ajmay81** [dftd4/dftd4#56]
More of an optimization than a bug, I guess if possible some reordering to have all the `wp` first would avoid this

```
class_param.f90(36): warning #6379: The structure contains one or more misaligned fields.   [DFTD_PARAMETER]
class_set.f90(32): warning #6380: The structure length is not a multiple of its largest element; could create misalignments for arrays of this type.   [DFTD_OPTIONS]
```

**Comment by awvwgk** [comment]
Just to note, reordering would result at least in ABI breakage.

Also, those parameters are usually not used in arrays, but I'm fine with adjusting this to account for the warnings.

**Comment by awvwgk** [comment]
This is also resolved with v3.0.0
